src/coord/src/coord.rs

```diff
@@ -1770,8 +1582,10 @@ impl Coordinator {
         self.drop_sinks(drop_sinks).await;
 
         // Allow compaction of sources from this transaction.
-        self.txn_reads.remove(&session.conn_id());
-
+        // TODO: Remove the above once this is confirmed to replace it.
```

Comment:
Confirm that the above has been removed, and then trash the comment.

Reply:
What is "the above" here?

Reply:
The code that got deleted. It was a comment I dropped in when I put `_neu` stuff to remind myself what needed clean-up.